CLIMBING.ipynb

```diff
@@ -30,77 +30,46 @@
     "import pandas as pd\n",
     "import matplotlib.pyplot as plt\n",
     "\n",
-    "def bouldering():\n",
-    "    climbers = pd.read_csv('bouldering_data.csv')\n",
-    "    df = climbers[['Age', 'Bouldering Grade', 'Years Climbing']]\n",
+    "# Retrieving data from CSV files\n",
+    "bouldering = pd.read_csv('bouldering_data.csv')\n",
+    "toprope = pd.read_csv('toprope_data.csv')\n",
+    "lead = pd.read_csv('leadclimb_data.csv')\n",
     "\n",
-    "    #create seaborn heatmap, assign axis labels\n",
+    "def climbing(data_file, x_axis_name, color):\n",
+    "    '''Allows you to plug seperate values for particular dataframe, name your bottom x axis\\' \n",
```

Comment:
It is nice that you have a `docstring` 
An improvement would be to explain the input arguments and the output.